### PR TITLE
DPC-813: Improve log format

### PR DESCRIPTION
**Pull request by embh**
**Why**
Logs are difficult to scan and search due to the current format, which complicates debugging.

**What Changed**
This branch adds `dropwizard-json-logging` to format log output as JSON. General logging is configured in `server.conf`, and request logs are configured in individual applications' `application.conf` files.

Screen Shot 2020-01-16 at 3 53 37 PM
Screen Shot 2020-01-16 at 4 19 43 PM
Screen Shot 2020-01-16 at 4 08 22 PM

**Tickets closed**:
[DPC-813](https://jiraent.cms.gov/browse/DPC-813)

**Future Work**
Additional logging improvements will be made with [DPC-835](https://jiraent.cms.gov/browse/DPC-835).

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-api/src/main/resources/application.conf [hunk 1]
```diff
@@ -15,10 +15,15 @@ dpc.api {
     }
 
     server.requestLog.appenders = [{
-        type: "console"
+        type = console
+        timeZone = UTC
```
Comment (RickHawesUSDS) [inline dpc-api/src/main/resources/application.conf]: Good. Should use UTC.

File: dpc-api/src/main/resources/application.conf [hunk 2]
```diff
@@ -15,10 +15,15 @@ dpc.api {
...
         filterFactories =[{
-            type: token-filter-factory
+            type = token-filter-factory
         }]
+        layout {
+          type = access-json
+          timestampFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+        }
     }]
```
Comment (RickHawesUSDS) [inline dpc-api/src/main/resources/application.conf]: A question... I'm wondering if this will include `x-request-id` and `authorization`.

**Comment by embh** [comment]
> Looks good. One question/suggestion. Do we need to do the same for the web site?

Definitely. That's included in DPC-835. Started a conversation with @switzersc-usds and @Sun-Mountain about structured logging for dpc-web.

File: dpc-api/src/main/resources/application.conf (hunk 2 above)
Comment (embh) [reply dpc-api/src/main/resources/application.conf]: Headers aren't included by default, but we can specify any that we need to keep track of. Configuration options: https://www.dropwizard.io/en/stable/manual/configuration.html#json-access-log-layout

**Comment by switzersc-usds** [comment]
Yeah, we should be able to do something similar to this with the Rails app, at least for HTTP request logging.

**Pull request merged**